### PR TITLE
Add override to the troubleshooting guide

### PR DESCRIPTION
**Pull request by bszwarc**
**Description**

Changes proposed in this pull request:

- Described how to add the override to change the trace sampling value.

**Related issue(s)**
See also #4103

### CODE REVIEW

File: docs/tracing/10-01-traces-troubleshooting.md [hunk 1]
```diff
@@ -5,14 +5,41 @@ type: Troubleshooting
...
+EOF
 ```
 
-Set the **PILOT_TRACE_SAMPLING** environment variable to a desired value, such as `60`.
-
-> **NOTE**: Using a very high value may affect Jaeger and Istio's performance and stability. Hence increasing the memory limits of Jaeger's deployment is needed.
+2. Proceed with the installation. Once the installation starts, the Kyma Operator will generate the override based on the ConfigMap and set the value of **PILOT_TRACE_SAMPLING** to `60`.
 
-Read the [Istio docs](https://istio.io/docs/tasks/observability/distributed-tracing/overview/#trace-sampling) for more information. 
+    >**NOTE:** If you use the override at runtime, run the following command to trigger the installation:
```
Comment (piotrmsc) [inline docs/tracing/10-01-traces-troubleshooting.md]:
```suggestion
    >**NOTE:** If you use the override at runtime, run the following command to trigger the update:
```

File: docs/tracing/10-01-traces-troubleshooting.md [hunk 2]
```diff
@@ -5,14 +5,41 @@ type: Troubleshooting
 
 ## Jaeger shows only a few traces
 
-Istio Pilot sets the trace sampling rate at `1`, where `100` is the maximum value. This means that only 1 out of 100 requests is sent to Jaeger for trace recording. To change this system behavior, run:
+By default, the **PILOT_TRACE_SAMPLING** value in the [IstioControlPlane](https://istio.io/docs/reference/config/istio.operator.v1alpha12.pb/) is set to `1`, where `100` is the maximum value. This means that only 1 out of 100 requests is sent to Jaeger for trace recording.
+To change this system behavior, [override](/root/kyma/#configuration-helm-overrides-for-kyma-installation) the existing configuration with a custom control plane definition.
+
+1. Add and apply a ConfigMap in the `kyma-installer` Namespace in which you set the value for the **PILOT_TRACE_SAMPLING** attribute to `60`.
```
Comment (piotrmsc) [inline docs/tracing/10-01-traces-troubleshooting.md]: Thinking out loud, shouldn't this be also part of production profile for monitoring?

File: docs/tracing/10-01-traces-troubleshooting.md (hunk 1 above)
Comment (bszwarc) [reply docs/tracing/10-01-traces-troubleshooting.md]: 👍

File: docs/tracing/10-01-traces-troubleshooting.md (hunk 2 above)
Comment (bszwarc) [reply docs/tracing/10-01-traces-troubleshooting.md]: I will talk to @a-thaler  about it :)

File: docs/tracing/10-01-traces-troubleshooting.md [hunk 3]
```diff
@@ -5,14 +5,41 @@ type: Troubleshooting
 
 ## Jaeger shows only a few traces
```
Comment (kazydek) [inline docs/tracing/10-01-traces-troubleshooting.md]: As agreed, that should be the title :)

File: docs/tracing/10-01-traces-troubleshooting.md [hunk 4]
```diff
@@ -5,14 +5,41 @@ type: Troubleshooting
...
+EOF
 ```
 
-Set the **PILOT_TRACE_SAMPLING** environment variable to a desired value, such as `60`.
-
-> **NOTE**: Using a very high value may affect Jaeger and Istio's performance and stability. Hence increasing the memory limits of Jaeger's deployment is needed.
+2. Proceed with the installation. Once the installation starts, the Kyma Operator will generate the override based on the ConfigMap and set the value of **PILOT_TRACE_SAMPLING** to `60`.
 
-Read the [Istio docs](https://istio.io/docs/tasks/observability/distributed-tracing/overview/#trace-sampling) for more information. 
+    >**NOTE:** If you use the override at runtime, run the following command to trigger the update:
```
Comment (kazydek) [inline docs/tracing/10-01-traces-troubleshooting.md]:
```suggestion
    >**NOTE:** If you use the override in the runtime, run the following command to trigger the update:
```

**Pull request merged**